test/test_utils/__init__.py

```diff
@@ -57,6 +57,12 @@ def is_tf2(image_uri):
     return bool(re.search(r'2\.\d+\.\d+', image_uri))
 
 
+def is_tf22(image_uri):
```

Comment:
Shouldn't the test be applicable for future TF versions as well?

Reply:
It might also apply for other TF2.X images(might including previous version TF2.0 2.1). Exploring. Will  extend the condition afterwards